<?xml version="1.0"?>
<launch>
  <rosparam file="$(find roberto_hw_interface)/config/controllers.yaml" command="load"/>
  <rosparam file="$(find roberto_hw_interface)/config/joint_limits.yaml" command="load"/>

  <node name="RobertoHardwareInterface" pkg="roberto_hw_interface" type="roberto_hw">
      <param name="drive_kP"    value="0.1"/>
      <param name="drive_kI"    value="0.002"/>
      <param name="drive_kD"    value="5.0"/>
      <param name="drive_kF"    value="0.035"/>
      <param name="drive_ramp"  value="1"/>
  </node>

  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
        args="
            /Roberto/joints_update
            /Roberto/auger_controller
            /Roberto/actuator_pos_controller
            /Roberto/bscrew_pos_controller
            /Roberto/left_wheel_controller
            /Roberto/right_wheel_controller"> 
  </node>
  
</launch>